cmd/set.go

```diff
@@ -45,48 +53,126 @@ func init() {
 }
 
 func setCmd(ctx *cli.Context) error {
-	key, value, errMsg := parseSetArgs(ctx.Args())
-	if errMsg != "" {
-		return errs.NewUsageExitError(errMsg, ctx)
-	}
-
-	cred, err := setCredential(ctx, key, func() *apitypes.CredentialValue {
-		return apitypes.NewStringCredentialValue(value)
-	})
+	args := ctx.Args()
+	secrets, err := parseSetSecrets(args)
 
 	if err != nil {
-		return errs.NewErrorExitError("Could not set credential.", err)
+		return errs.NewUsageExitError(err.Error(), ctx)
 	}
 
-	name := (*cred.Body).GetName()
-	pe := (*cred.Body).GetPathExp()
-	fmt.Printf("\nCredential %s has been set at %s/%s\n", name, pe, name)
+	for _, secret := range secrets {
+		cred, err := setCredential(ctx, secret.key, func() *apitypes.CredentialValue {
+			return apitypes.NewStringCredentialValue(secret.value)
+		})
+
+		if err != nil {
+			return errs.NewErrorExitError("Could not set credential.", err)
+		}
+
+		name := (*cred.Body).GetName()
+		pe := (*cred.Body).GetPathExp()
+		fmt.Printf("\nCredential %s has been set at %s/%s\n", name, pe, name)
+	}
 
 	hints.Display(hints.View, hints.Run)
 	return nil
 }
 
-func parseSetArgs(args []string) (key string, value string, errMsg string) {
-	// If there is only one argument, try to parse using env var syntax: KEY=value
+// parseSetSecrets returns a list of secret pairs either from the command line
+// arguments or from the standard input. It returns an error if there is a
+// problem parsing secrets or stdin fails to read.
+func parseSetSecrets(args []string) ([]secretPair, error) {
+	if len(args) > 0 {
+		secret, err := parseSetArgs(args)
+		return []secretPair{secret}, err
+	}
```

Comment:
Would it make sense to introduce this behavior as a separate command?

ie. `torus import [filename]` (or naked to accept stdin)